packages/@tailwindcss-upgrade/src/template/codemods/modernize-arbitrary-values.test.ts

```diff
@@ -31,6 +31,14 @@ test.each([
   ['group-[]:flex', 'in-[.group]:flex'],
   ['group-[]/name:flex', 'in-[.group\\/name]:flex'],
 
+  // There is no good equivalent for this one right now:
+  ['group-hover/[]:flex', 'group-hover/[]:flex'],
```

Comment:
The comment here is confusing in relation to this PR since the latter won't parse anymore so are we actually breaking this with no fallback? 🤔

Reply:
This is technically a breaking change because it did parse in v3, but as far as I can tell nobody uses that (because it doesn't really make sense)

This is what it produces in v3 today:
```css
.group\/\[\]:hover .group-hover\/\[\]\:flex {
  display: flex;
}
```

https://sourcegraph.com/search?q=context:global+/%5B%5D+repohasfile:tailwind.config&patternType=keyword&sm=0